monitoring/defender/autotask/on_module_factory_events.js

```diff
@@ -0,0 +1,1014 @@
+    type: "function",
+  },
+  {
+    inputs: [
+      {
+        internalType: "uint32",
+        name: "timeout",
+        type: "uint32",
+      },
+    ],
+    name: "setQuestionTimeout",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      {
+        internalType: "address",
+        name: "_target",
+        type: "address",
+      },
+    ],
+    name: "setTarget",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      {
+        internalType: "uint256",
+        name: "templateId",
+        type: "uint256",
+      },
+    ],
+    name: "setTemplate",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      {
+        internalType: "bytes",
+        name: "initParams",
+        type: "bytes",
+      },
+    ],
+    name: "setUp",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [],
+    name: "target",
+    outputs: [
+      {
+        internalType: "address",
+        name: "",
+        type: "address",
+      },
+    ],
+    stateMutability: "view",
+    type: "function",
+  },
+  {
+    inputs: [],
+    name: "template",
+    outputs: [
+      {
+        internalType: "uint256",
+        name: "",
+        type: "uint256",
+      },
+    ],
+    stateMutability: "view",
+    type: "function",
+  },
+  {
+    inputs: [
+      {
+        internalType: "address",
+        name: "newOwner",
+        type: "address",
+      },
+    ],
+    name: "transferOwnership",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+];
+
+exports.handler = async function (event) {
+  console.log(
+    "\n\n/************************** EVENT *****************************/"
+  );
+  console.log(JSON.stringify(event));
+  console.log(
+    "/**********************************************************/\n\n"
+  );
+  if (event && event.request && event.request.body) {
+    // variables from autotask creation
+    const nodeProvider = "{{nodeProvider}}";
+    const discordWebHookUrl = "{{discordWebHookUrl}}";
+    const transaction = event.request.body.transaction;
+    const chainId = event.request.body.sentinel.chainId;
+    const txHash = transaction.transactionHash;
+    const provider = new ethers.providers.JsonRpcProvider(nodeProvider);
+    const tx = await provider.getTransactionReceipt(txHash);
+    const utils = ethers.utils;
+    console.log(
+      "\n\n/*********************** TRANSACTION **************************/"
+    );
+    const sig = "ModuleProxyCreation(address,address)"; //ModuleProxyCreation (index_topic_1 address proxy, index_topic_2 address masterCopy)
+    const bytes = utils.toUtf8Bytes(sig);
+    const keccak = utils.keccak256(bytes);
+    let proxyTopic = "";
+    const logs = tx.logs || [];
+    console.log("logs", JSON.stringify(logs));
+    if (logs) {
+      logs.forEach((logs) => {
```

Comment:
I would recommend reaming the inner `logs` to, for instance, `log` (variable shadowing can make it hard to reason about the code).